### DISCUSSION TITLE
Can someone please help me install control net or answer some basic questions?

### DISCUSSION THREAD

**Discussion by viperwasp** [lllyasviel/stable-diffusion-webui-forge#2149] [General]
I have not installed control net since the early days of action 1111. I only just started to use Forge UI. I have only used ComfyUI a little. 

Here are my main questions. 
1. I have a full updated ForgeUI. Where do I go to get started downloaded Control net? I am mostly into the feature that makes it so it like puts line in an image so it will generate images or do image to image and make faces or other objects very similar to the controlnet image etc. Not poses but the whole image is like traced. But I don't care if I have to install other control nets with it that's fine. 

2. Do I need a seperate control net install from SD 1 to XL to Flux? How many total do I need? 

3. I guess that's it? I don't know where to go most guides are for Action 1111 or ComfyUI? 
Thank you.

**Comment by mold0101** [comment]
1. Installation
   - ControlNet is included in Forge, you just need to download the model files
   - Visit Civitai.com and search for "ControlNet" to find comprehensive model collections
   - Place downloaded models in the models/controlnet folder

2. Version Compatibility
   - Always match ControlNet version with corresponding Stable Diffusion checkpoint version (es,  ControlNet 1.5 models must be paired with SD 1.5 checkpoints and so on)

3. Learning Resources
   - Most 1111 guides are applicable to Forge when you grasp the basic principles
   
Hope this helps!

**Reply by viperwasp** [reply]
Yes that helps a lot thank you!
